src/common/webrequest_curl.cpp

```diff
@@ -197,13 +314,15 @@ wxWebRequestCURL::wxWebRequestCURL(wxWebSession & session,
     // Enable all supported authentication methods
     curl_easy_setopt(m_handle, CURLOPT_HTTPAUTH, CURLAUTH_ANY);
     curl_easy_setopt(m_handle, CURLOPT_PROXYAUTH, CURLAUTH_ANY);
+    // Block curl from sending messages to stderr.
+    curl_easy_setopt(m_handle, CURLOPT_STDERR, m_devNull.fp());
```

Comment:
```suggestion
```

Reply:
This has been removed since we're no longer using CURLOPT_STDERR.